core/interfaces.go

```diff
@@ -137,6 +143,7 @@ type Storage interface {
 	FindSentNudge(nudgeID string, userID string, netID string, criteriaHash uint32, mode string) (*model.SentNudge, error)
 	FindSentNudges(nudgeID *string, userID *string, netID *string, criteriaHash *[]uint32, mode *string) ([]model.SentNudge, error)
 	DeleteSentNudges(ids []string, mode string) error
+	FindNudgesProcess(limit int, offset int) ([]model.NudgesProcess, error)
```

Comment:
Order this next to NudgesProcesses entity but not to SentNudges ones. Also, the name should be `FindNudgesProcesses`